op-guide/migration-overview.md

```diff
@@ -1,58 +1,50 @@
 ---
 title: Migration Overview
-summary: Learn how to migrate data from MySQL to TiDB.
+summary: Learn how to migrate data into TiDB
 category: operations
 ---
 
 # Migration Overview
 
-## Overview
+This document describes scenarios for migrating data into TiDB from either MySQL or data source stored in CSV format.
 
-This document describes how to migrate data from MySQL to TiDB in detail.
+## Tools overview
 
-See the following for the assumed MySQL and TiDB server information:
+Migrations will often make use of the following tools. For a brief overview of their usage:
 
-|Name|Address|Port|User|Password|
-|----|-------|----|----|--------|
-|MySQL|127.0.0.1|3306|root|* |
-|TiDB|127.0.0.1|4000|root|* |
+- [`mydumper`](../tools/mydumper.md) exports data from MySQL. It is recommended over using mysqldump.
```

Comment:
```suggestion
- [`mydumper`](../tools/mydumper.md) exports data from MySQL. It is recommended over using [mysqldump](https://github.com/DSB/MySQLDumper).
```

Reply:
mysqldump is the included program with mysql.  It is not the same as mysqldumper.

Reply:
Ok, then we need to clarify it.

Reply:
Discussed on chat: it is a common program, all mysql users will know it.